src/main/java/org/folio/eusage/reports/api/EusageReportsApi.java

```diff
@@ -38,14 +41,18 @@
 import org.folio.tlib.postgres.TenantPgPool;
 
 public class EusageReportsApi implements RouterCreator, TenantInitHooks {
-  private final Logger log = LogManager.getLogger(EusageReportsApi.class);
+  private static final Logger log = LogManager.getLogger(EusageReportsApi.class);
 
   private WebClient webClient;
 
   static String titleEntriesTable(TenantPgPool pool) {
     return pool.getSchema() + ".te_table";
   }
 
+  static String packageEntriesTable(TenantPgPool pool) {
+    return pool.getSchema() + ".pe_table";
```

Comment:
Why do the table names have the `_table` suffix that is neither needed nor add information to the name?

Why do the tables names consist of abbreviations like `pe` that are incomprehensible?

Reply:
ok. will rename.